investigator/investigator/missing_package/investigate_missing_package.py

```diff
@@ -37,30 +37,11 @@
 @register_handler(MissingPackageMessage().topic_name, ["v1"])
 async def parse_missing_package(package: Dict[str, Any], openshift: OpenShift, graph: GraphDatabase):
     """Process a missing package message from package-update producer."""
-    repositories = graph.get_adviser_run_origins_all(
-        index_url=package["index_url"], package_name=package["package_name"], count=None, distinct=True,
-    )
-
-    issue_title = f"Missing package {package['package_name']} on {package['index_url']}"
-
-    def issue_body():
-        return "Automated message from package change detected by thoth.package-update"
-
-    kebechet_wf_scheduled = 0
-    for repo in repositories:
-        gitservice_repo = git_source_from_url(repo)
-        requirements = re.split("\n| ", gitservice_repo.service.get_project().get_file_content("Pipfile"))
-
-        if package["package_name"] in requirements:
-            gitservice_repo.open_issue_if_not_exist(issue_title, issue_body)
-        else:
-            is_scheduled = await schedule_kebechet_run_url(
-                openshift=openshift, repo=repo, gitservice_repo_name=gitservice_repo.service_type.name
-            )
-            kebechet_wf_scheduled += is_scheduled
+    # TODO call kebechet with the missing package name, which would call kebechet on the individual
```

Comment:
do we do need `graph` in the method here?

Reply:
Removed these for now, I am not sure what we want to do apart from scheduling so had them there.

Reply:
In theory we want investigator really to do the minimum, aka schedling things, graph connections should be handled by components directly as much as possible